ip_map_branch.txt

```diff
@@ -28,17 +28,17 @@ seven_c	https://github.com/openemr/openemr.git	rel-501	0	1	0	0	demo_5_0_0_5.sql
 seven_d	https://github.com/openemr/openemr.git	master	0	0	1	0	0	0	1	seven.openemr.io/d	hey	branch	0	0	0	0	OPEN
 seven_e	https://github.com/openemr/openemr.git	rel-502	0	0	1	0	0	0	1	seven.openemr.io/e	hey	branch	0	1	0	0	Public OpenEMR 5.0.2 Development Demo on Ubuntu 18.04
 seven_f	https://github.com/openemr/openemr.git	rel-502	0	0	0	0	demo_5_0_0_5.sql	0	1	seven.openemr.io/f	hey	branch	5.0.0	1	0	0	Public OpenEMR 5.0.2 Development Demo on Ubuntu 18.04 With Demo Data
-eight	https://github.com/openemr/openemr.git	master	0	1	0	0	0	0	1	eight.openemr.io	hey	branch	0	2	0	0	Public OpenEMR Development Demo on Alpine 3.12
-eight_a	https://github.com/openemr/openemr.git	master	0	1	0	0	demo_5_0_0_5.sql	0	1	eight.openemr.io/a	hey	branch	5.0.0	2	0	0	Public OpenEMR Development Demo on Alpine 3.12 With Demo Data
+eight	https://github.com/openemr/openemr.git	master	0	1	0	0	0	0	1	eight.openemr.io	hey	branch	0	2	0	0	OPEN
+eight_a	https://github.com/openemr/openemr.git	master	0	1	0	0	demo_5_0_0_5.sql	0	1	eight.openemr.io/a	hey	branch	5.0.0	2	0	0	OPEN
```

Comment:
Would prob make description `OPEN - Alpine 3.12` (just so we know what it is without needing to look elsewhere). And should then remove these 2 demos listing from here: 
https://www.open-emr.org/wiki/index.php/Development_Demo#Alternate_Demo_.28Alpine_3.12_-_uses_php_7.3.29

(note that having demos with outdated php is not that big of deal and then at least get to test that it does not work; ie. it properly shows the message that the php version is to low; note that there are a bunch of lower version alpines on that demo page that have not yet been "reprocessed")

Reply:
Actually, disregard the `OPEN` changes since not doing elsewhere and realized easy to figure this out if needed.

Reply:
To clarify, just using `OPEN` is fine :)